languages/common-lisp/exercises/concept/sameness/.docs/hints.md

```diff
@@ -0,0 +1,23 @@
+## The Maze of Object Equality
```

Comment:
This is looking very good! I've messed it up in my PR as well, but I think the items under the headings should be bulleted, the titles preceded by their number, and the titles in regular sentence case.

Erik linked me here: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd

And there is an example here: https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/floating-point-numbers/.docs/hints.md

Reply:
Ah, thanks  - I'll add numbers and fix casing.